cli/cmd/auth.go

```diff
@@ -111,12 +117,12 @@ keptn auth --skip-namespace-listing # To skip the listing of namespaces and use
 				OauthClientSecret: *authParams.oauthClientSecret,
 				OauthScopes:       *authParams.oauthScopes,
 			}
-			if err := oauth.Auth(clientValues); err != nil {
+			if err := oauth.Auth(clientValues, port); err != nil {
```

Comment:
I'd suggest adding `port` to the `OauthClientValues` object. It belongs semantically to that struct; also you don't need to change so many function signatures :)